sdk/python/lib/pulumi/provider/experimental/ComponentProvider.md

```diff
@@ -177,7 +177,7 @@ python -m venv venv
 source venv/*/activate
 pip install -r requirements.txt
 # Get the schema
-pulumi package get-schema .
+pulumi package get-schema ./
```

Comment:
THere's a bug and it won't work with `.` https://github.com/pulumi/pulumi/issues/18529